doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
Mmm, I am not sure, especially when I read "we assign "rank" to all siblings among test suite children." So there is still an ordering following the tree structure, while it should not be. This is why I am also proposing leaving out the (1).